README.md

```diff
@@ -649,6 +650,14 @@ while IFS= read -r line || [ -n "$line" ]; do
 done < "file"
 ```
 
+## Loop over the output of a command
+
+```shell
+command|while IFS= read -r line || [ -n "$line" ]; do
+    printf '%s\n' "$line"
+done
+```
+
```

Comment:
```suggestion
## Loop over the output of a command

```shell
while IFS= read -r line || [ -n "$line" ]; do
    printf '%s\n' "$line"
done <<EOF
$(command)
EOF
```

```

Reply:
this technically works the same way, but not with a subshell